fence/agents/compute/fence_compute.py

```diff
@@ -218,72 +218,71 @@ def fix_domain(options):
 	if nova:
 		# Find it in nova
 
-		hypervisors = nova.hypervisors.list()
-		for hypervisor in hypervisors:
-			shorthost = hypervisor.hypervisor_hostname.split('.')[0]
+		services = nova.services.list(binary="nova-compute")
```

Comment:
This is explained in the commit message: https://github.com/ClusterLabs/fence-agents/pull/80/commits/5cf13d5866d0541281c914ab4edf42cd09d5462b